CHANGES.rst

```diff
@@ -1,3 +1,9 @@
+1.12 (Apr 25, 2022)
```

Comment:
@JasperCraeghs 
`doxylink` did not support parallel build, it was tested for the compatibility and the feature enabled. Therefore I consider this as a feature rather than a bugfix. Also it was labeled as `enhacement.`
Any specific reason why you see this differently?

Reply:
You're right. I didn't realize that 1.12 was already released as it is not listed in https://github.com/sphinx-contrib/doxylink/releases

Reply:
The new CI script did not run fully the last release so the GitHub release was not auto-created. I have now manually added the release item. It should happen automatically the next time.